test/e2e/save_test.go

```diff
@@ -75,7 +75,7 @@ var _ = Describe("Podman save", func() {
 		Expect(save).To(ExitWithError())
 	})
 
-	It("podman save to directory with oci format", func() {
+	It("podman save and load to directory with oci format", func() {
```

Comment:
Looking at the test failures:
- At least going by descriptions without digging into PR history, it’s not clear that all of them wanted to use the OCI directory format; maybe they intended to use `docker-dir`.
- `podman load` does not implement `oci-archive` (https://github.com/containers/libpod/blob/091b9b1163ecbc3e64a9152b5e57e969b4475c2a/libpod/runtime_img.go#L257 ), `podman pull oci-archive:…` might work in the meantime.

Reply:
It’s not entirely clear that `podman load` should try all of these formats — but probably so (and leaving `podman pull` for the remote pulls and the explicit transport:… uses). It might be worth checking whether the input path is a directory instead of trying all four formats every time.

Reply:
The error I am seeing is a lack of manifest.json file?
```
# /bin/podman save --format docker-dir -o /tmp/docker-dir alpine
# ./bin/podman save --format oci-dir -o /tmp/oci-dir alpine
# ls /tmp/docker-dir
50644c29ef5a27c9a40c393a73ece2479de78325cae7d762ef3cdc19bf42dd0a  blobs       manifest.json  version
a24bb4013296f61e89ba57005a7b3e52274d8edd3ae2077d04395f806b63d83e  index.json  oci-layout
# ls /tmp/oci-dir
blobs  index.json  oci-layout
```
Should there be a manifest.json file in the oci-dir?

Reply:
@mtrmac ^^

Reply:
There shouldn’t be a `manifest.json`. The missing `manifest.json` error is reported by a `dir:` source — as mentioned above, `podman load` tries to use `dir:` but not `oci:` (I incorrectly said `oci-archive:`, which _is_ tried).